app/javascript/dashboard/modules/widget-preview/components/Widget.vue

```diff
@@ -0,0 +1,293 @@
+<template>
+  <div class="widget-wrapper">
+    <div class="header-wrapper">
+      <div class="header-branding">
+        <div class="header">
+          <img
+            v-if="logo"
+            :src="logo"
+            class="logo"
+            :class="{ small: !isExpanded }"
+          />
+          <div v-if="!isExpanded">
+            <div class="title-block text-base font-medium">
+              <span class="mr-1">{{ websiteName }}</span>
+              <div v-if="isOnline" class="online-dot"></div>
+            </div>
+            <div class="text-xs mt-1 text-black-700">
+              {{ responseTime }}
+            </div>
+          </div>
+        </div>
+        <div v-if="isExpanded" class="header-expanded">
+          <h2 class="text-slate-900 mt-6 text-4xl mb-3 font-normal">
+            {{ welcomeHeading }}
+          </h2>
+          <p class="text-lg text-black-700 leading-normal">
+            {{ welcomeTagLine }}
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="conversation--container">
+      <div class="conversation-wrap">
+        <div class="message-wrap">
+          <div class="user-message-wrap">
+            <div class="user-message">
+              <div class="message-wrap">
+                <div class="chat-bubble user">
+                  <p>Hello</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="agent-message-wrap">
+          <div class="agent-message">
+            <div class="avatar-wrap"></div>
+            <div class="message-wrap">
+              <div class="chat-bubble agent">
+                <div class="message-content">
+                  <p>Hello</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="footer-wrap">
+      <div class="input-wrap">
+        <footer class="footer">
+          <div class="chat-message--input">
+            <text-area
```

Comment:
We could use a dummy `div` or disable the `textarea` so that css minimal.

Reply:
Updated to div as discussed